sample/TimerTrigger-CSharp/function.json

```diff
@@ -3,7 +3,7 @@
     "input": [
       {
         "type": "timerTrigger",
-        "schedule": "0 * * * * *",
+        "schedule": "0/10 * * * * *",
```

Comment:
nit: recommend leaving this at once per minute - otherwise the sample app is way to chatty